bq_dts/helpers.py

```diff
@@ -162,6 +162,8 @@ def normalize_transfer_run(transfer_run, integer_params=None):
     #   param_name: {string_value: '1901-01-01'}
     # TO
     #   param_name: '1901-01-01'
+    if not 'params' in transfer_run or not out_transfer_run['params']:
```

Comment:
Will it be the same result and easier to read? That way, the None check in protobuf_struct_to_python_dict() can be skipped.

    if out_transfer_run['params'] is not None:
        out_params = protobuf_struct_to_python_dict(out_transfer_run['params'])

Reply:
This was handling two cases. First when the params was not in the transfer_run and when the params was null. I haven't seen the first in transfer_runs from the server, but did when I hand coded it for testing.